### DISCUSSION TITLE
Compiling on M1

### DISCUSSION THREAD

**Discussion by dim13** [OpenI6X/opentx#407] [Show and tell]
# Arm GCC

Instead of `brew install arm-none-eabi-gcc` use `brew install --cask gcc-arm-embedded`

Add `set(APPLE FALSE)` to main `CMakeList.txt`

It would be nice to add it to the Wiki.

**Comment by ajjjjjjjj** [comment]
Could you provide complete documentation block to add to Development guide under Compile section?
